src/Symfony/Component/Console/Application.php

```diff
@@ -753,12 +753,20 @@ protected function doRenderException(\Exception $e, OutputInterface $output)
         do {
             $message = trim($e->getMessage());
             if ('' === $message || OutputInterface::VERBOSITY_VERBOSE <= $output->getVerbosity()) {
-                $title = sprintf('  [%s%s]  ', \get_class($e), 0 !== ($code = $e->getCode()) ? ' ('.$code.')' : '');
+                $class = \get_class($e);
+                $class = 'c' === $class[0] && 0 === strpos($class, "class@anonymous\0") ? get_parent_class($class).'@anonymous' : $class;
```

Comment:
I'm curious about `\0`, I know it represents a null byte, but that's the first time I see it in PHP code.
Does it improve perf or something like that? 🤔

Reply:
You cannot print a null char using `null` as this would result in an empty string 😃

Reply:
That's desired, see https://github.com/symfony/symfony/pull/28218#discussion_r211523507